lib/checkother.cpp

```diff
@@ -2510,8 +2511,10 @@ void CheckOther::checkUnusedLabel()
                 tok = tok->scope()->classEnd;
 
             if (Token::Match(tok, "{|}|; %name% :") && tok->strAt(1) != "default") {
-                if (!Token::findsimplematch(scope->classStart->next(), ("goto " + tok->strAt(1)).c_str(), scope->classEnd->previous()))
-                    unusedLabelError(tok->next(), tok->next()->scope()->type == Scope::eSwitch);
+                if (!Token::findsimplematch(scope->classStart->next(), ("goto " + tok->strAt(1)).c_str(), scope->classEnd->previous())) {
```

Comment:
Here order of parameters evaluation doesn't matter because evaluating either of parameters doesn't affect evaluation of others. Technically it's an FP.